index.md

```diff
@@ -11,7 +11,7 @@ developed by the
 
 The current specification is based on *tracking by detection* techniques (see computational workflow in the following figure) and the data produced therein:
 
-![Workflow](images/Workflow.png){:class="img-responsive"}
+![workflow](images/workflow.png){:class="img-responsive"}
```

Comment:
An now the image seems to be broken in https://cellmigstandorg.github.io/Tracks/. One last rendering fix PR is required.